test/cctest/test_inspector_socket.cc

```diff
@@ -298,7 +298,7 @@ static void really_close(uv_handle_t* handle) {
 // Called when the test leaves inspector socket in active state
 static void manual_inspector_socket_cleanup() {
   EXPECT_EQ(0, uv_is_active(
-                   reinterpret_cast<uv_handle_t*>(&inspector.client)));
+            reinterpret_cast<uv_handle_t*>(&inspector.client)));
```

Comment:
This looked okay to me? `reinterpret_cast<…>()` is an argument to `uv_is_active`, not `EXPECT_EQ`